src/components/Message/Message.tsx

```diff
@@ -408,6 +408,13 @@ const MessageWithContext = <
   const doubleTapRef = useRef<TapGestureHandler>(null);
   const pressActive = useSharedValue(false);
   const scale = useSharedValue(1);
+  useEffect(
+    () => () => {
```

Comment:
a function -> function?

Reply:
just comment about this about it being a cleanup function only for a nav based issue